src/index.ts

```diff
@@ -57,6 +60,7 @@ export = function (schema: Mongoose.Schema, options: PluginOptions) {
     if (this.isNew) {
       // console.log('[new]');
       this[VERSION] = 1;
+      delete this[CHANGED_BY];
```

Comment:
not sure why this is needed, but if it is needed, shouldnt CHANGED_AT also be added here?

Reply:
you actually explained this one on Slack - to prevent it from being in the returned document, right..
Still, should CHAGNED_AT be here as well then?

Reply:
no, changed at is added only into vermongo model (which has default value new Date) so this is not being passed from execution code, we can talk tomorrow and I can explain/show